test/e2e/flowaggregator_test.go

```diff
@@ -216,8 +215,8 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 
 	// Polling to make sure all the data records corresponding to the iperf flow
 	// are received.
-	err = wait.Poll(250*time.Millisecond, collectorCheckTimeout, func() (bool, error) {
-		rc, collectorOutput, _, err := provider.RunCommandOnNode(controlPlaneNodeName(), fmt.Sprintf("kubectl logs --since=%v ipfix-collector -n antrea-test", time.Since(timeStart).String()))
+	err = wait.PollImmediate(500*time.Millisecond, flowAggregatorExportInterval, func() (bool, error) {
```

Comment:
PR#1904 changed wait condition in line 224 from `return strings.Contains(collectorOutput, srcIP) && strings.Contains(collectorOutput, dstIP), nil` to `return strings.Count(collectorOutput, srcIP) >= expectedNumDataRecords && strings.Count(collectorOutput, dstIP) >= expectedNumDataRecords, nil` to avoid the case when we will only receive one record. So here we can use `flowAggregatorExportInterval * 2`.

Reply:
Polling happens after iperf runtime, which is 10s currently (`flowAggregatorExportInterval * 2`). The time out will capture the final records if there are any, so I feel one cycle of `flowAggregatorExportInterval` is enough. What do you think?

Reply:
Makes sense to me. Thanks for clarifying.